src/main/java/com/denizenscript/depenizen/bukkit/commands/quests/QuestsCommand.java

```diff
@@ -116,84 +115,73 @@ public void execute(ScriptEntry scriptEntry) {
             Debug.report(scriptEntry, getName(), action, questId, stageNum, points, override_checks);
         }
         switch (Action.valueOf(action.asString().toUpperCase())) {
-            case ADD: {
+            case ADD -> {
                 if (questId != null) {
-                    for (Quest quest : quests.getQuests()) {
+                    for (Quest quest : quests.getLoadedQuests()) {
                         if (quest.getId().equals(questId.asString())) {
                             quests.getQuester(player.getPlayerEntity().getUniqueId()).takeQuest(quest, override_checks.asBoolean());
                             break;
                         }
                     }
-                }
-                else if (points != null && points.asInt() > 0) {
+                } else if (points != null && points.asInt() > 0) {

```

Comment:
This formatting should not have been changed

Reply:
Not sure how that happened, will fix